msg/PublishedTime.msg

```diff
@@ -0,0 +1,7 @@
+# The debug message which includes header and publishing timestamp
+
+# Message header time
+builtin_interfaces/Time header_stamp
```

Comment:
```suggestion
std_msgs/Header header
```

Refs:
- https://github.com/ros2/common_interfaces/tree/humble/std_msgs/msg
- https://github.com/ros2/message_filters/blob/humble/index.rst

Do the default message filters even work with this custom name and this time stamp?